src/components/NewPost/NewPost.tsx

```diff
@@ -0,0 +1,125 @@
+import React, { useState, useCallback, useMemo } from 'react';
+import Todo from '../../types/Todo';
+import usersFromServer from '../../api/users';
+import getUserById from '../../utils/getUserById';
+
+interface Props {
+  setTodos: React.Dispatch<React.SetStateAction<Todo[]>>;
+  todos: Todo[],
+}
+
+export const NewPost: React.FC<Props> = ({
+  setTodos,
+  todos,
+}) => {
+  const INITIAL_USER_ID = useMemo(() => '0', []);
```

Comment:
let's just create the simple constant and move out of the component

Reply:
I think it will be logical if it is in the component